docs/aws-sts-provisioning.md

```diff
@@ -0,0 +1,78 @@
+# Provisioning AWS STS Clusters
+
+It is possible to use Hive to provision clusters configured to use Amazon's Security Token Service, where cluster components use short lived credentials that are rotated frequently. This feature was added to the in-cluster Cloud Credential Operator in 4.7, see documentation [here](https://docs.openshift.com/container-platform/4.7/authentication/managing_cloud_provider_credentials/cco-mode-sts.html).
```

Comment:
I'd word it like: "This feature was added to the in-cluster OpenShift components in 4.7".